front/src/components/atoms/Popup.tsx

```diff
@@ -0,0 +1,58 @@
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
+import { Dialog, Transition } from "@headlessui/react";
+import { Fragment, ReactNode } from "react";
+import { ButtonIcon } from "./ButtonIcon";
+import { CardAtom } from "./CardAtom";
+
+type PopupProps = {
+    children: ReactNode;
+    open: boolean;
+    setOpen: (open: boolean) => void;
+};
+
+export const Popup = ({ children, open, setOpen }: PopupProps) => {
+    return (
+        <Transition.Root show={open} as={Fragment}>
+            <Dialog as="div" className="relative z-10" onClose={() => {}}>
```

Comment:
close is a different ticket im assuming?

Reply:
How do users close the dialog in this case?

Reply:
There is a button in the top right corner (see screenshot)